charts/home-assistant/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 0.116.1
 description: Home Assistant
 name: home-assistant
-version: 2.5.2
+version: 2.5.3
```

Comment:
It's a minor thing but do you mind bumping the minor version to 2.6.0 for this change?

Reply:
Fixed